examples/vector_indices/OpensearchDemo.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Using as a vector index.\n",
+    "\n",
+    "Elasticsearch only supports Lucene indices, so only Opensearch is supported."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "from os import getenv\n",
+    "from gpt_index import SimpleDirectoryReader\n",
```

Comment:
i know this is a bit confusing but we're doing name migrations to llama_index so let's use llama_index imports instead of gpt_index

Reply:
to test this you can try `LLAMA_INDEX_DIR=../llama_index GPT_INDEX_DIR=. sh create_llama_package.sh` from the root directory, and pip installing the generated llama_index build `pip install ../llama_index/dist/*`